docs/plugin-syntax-dynamic-import.md

```diff
@@ -4,6 +4,17 @@ title: @babel/plugin-syntax-dynamic-import
 sidebar_label: syntax-dynamic-import
 ---
 
+`@babel/plugin-syntax-dynamic-import` is needed to enable parsing for import().
+
+In Babel 7.8.0 it's enabled by default, so this plugin shouldn't be needed if you are using `@babel/core@>7.8.0`.
+
+If you want to transpile import():
```

Comment:
It reads a bit weird when this line starts with "if you want..." and two of the following items do too.

Maybe this can be,

> Some usage notes:

?

Reply:
👍